test/test_events.py

```diff
@@ -69,6 +69,11 @@ def mocked_events_ics(source: str = "uqcs") -> bytes:
     with open("test/test_events_events.ics", "rb") as events_file:
         return events_file.read()
 
+def mocked_get_august_2018_time():
+    """
+    Returns a fixed datetime with recurring events in the future.
```

Comment:
This function doesn't appear to actually be responsible for the recurring future events, maybe the description could be made clearer.